### ISSUE TITLE
Transfer from Nami to Kucoin pending

### ISSUE THREAD

**Issue by MadPhoenix** [Berry-Pool/nami-wallet#364]
Hello,
I did a transfer of 1000 ADA to Kucoin from Nami and it arrived after an hour without any issue

I did the same thing right after with 9000 ADA and the transaction seems to be still pending and still hasn't arrived after 16 hours, do I have to wait and if so how long ?

**Comment by alessandrokonrad** [comment]
> I did the same thing right after with 9000 ADA and the transaction seems to be still pending and still hasn't arrived after 16 hours, do I have to wait and if so how long ?

Do you see the transaction in the history tab of Nami or on cardanoscan.io?

**Comment by MadPhoenix** [comment]
Yes, it appears in the history transactions and have access to tx hash and stuff, it tells me 2600 confirmation, assurance high

**Comment by alessandrokonrad** [comment]
Then it's related to Kucoin. Please ask their support why you can't see it on the exchange yet.
